### ISSUE TITLE
Oid to Mib,use the command--mibdump.py  LSI-MegaRAID-SAS-MIB

### ISSUE THREAD

**Issue by wuyuwu0215** [etingof/pysmi#37]
Use midsump.py to convert LSI-MegaRAID-SAS-MIB.
 
  At first,I use the command--mibdump.py  LSI-MegaRAID-SAS-MIB.The mistake is “Duplicate symbol found: bbuFRU at MIB LSI-MegaRAID-SAS-MIB”.
  So I download the file,chang it to make it not a duplicate part.Then I use this command--mibdump.py --mib-source="http://192.168.1.170/" "LSI-MegaRAID-SAS1-MIB".That's why I can run the local modified file(LSI-MegaRAID-SAS-MIB).
  The new mistake appeared “no module "SNMPv2-SMI" in symbolTable at MIB LSI-MegaRAID-SAS-MIB”.

  I listened to some suggestions,
use the command--mibdump.py --mib-source=http://mibs.snmplabs.com/asn1 LSI-MegaRAID-SAS1-MIB.
  It was not go,the mistake of "Missing source MIBs: LSI-MegaRAID-SAS1-MIB".

**Comment by etingof** [comment]
Try adding two `--mib-source` options like this:

    mibdump.py --mib-source http://192.168.1.170/ --mib-source http://mibs.snmplabs.com/asn1 LSI-MegaRAID-SAS1-MIB

**Comment by wuyuwu0215** [comment]
I use the command:
mibdump.py --mib-source http://192.168.1.170/ --mib-source http://mibs.snmplabs.com/asn1 LSI-MegaRAID-SAS1-MIB
No successful conversion.Missing source MIBs: RFC-1212, RFC-1215, RFC1155-SMI, RFC1213-MIB, SNMPv2-CONF, SNMPv2-SMI, SNMPv2-TC.
I feel there's no way to get down.A little sad.

**Comment by wuyuwu0215** [comment]
The original name of the file is LSI-MegaRAID-SAS-MIB.In order to distinguish them，I rename i to LSI-MegaRAID-SAS1-MIB.
  I had also tried the command-- mibdump.py --mib-source http://192.168.1.170/ --mib-source http://mibs.snmplabs.com/asn1 LSI-MegaRAID-SAS-MIB.（rename it all to LSI-MegaRAID-SAS-MIB）
There's the same mistake.

**Comment by wuyuwu0215** [comment]
I tried it again.
`>mibdump.py --mib-source=http://192.168.1.170/ --mib-source=http://mibs.snmplabs.com/asn1 LSI-MegaRAID-SAS-MIB`

`Missing source MIBs: RFC-1212, RFC-1215, RFC1155-SMI, RFC1213-MIB, SNMPv2-CONF, SNMPv2-SMI, SNMPv2-TC
Ignored MIBs:
Failed MIBs: LSI-MegaRAID-SAS-MIB (no module "SNMPv2-SMI" in symbolTable at MIB LSI-MegaRAID-SAS-MIB)`

**Comment by etingof** [comment]
Perhaps you are using some old pysmi version. Try this command:

    mibdump.py --mib-source=http://192.168.1.170/ --mib-source=http://mibs.snmplabs.com/asn1/@mib@ LSI-MegaRAID-SAS-MIB

**Comment by wuyuwu0215** [comment]
Oh！Yes！Thank you very much!It has been successfully converted！Cool！

**Comment by YorkZ** [comment]
> Perhaps you are using some old pysmi version. Try this command:
> 
> ```
> mibdump.py --mib-source=http://192.168.1.170/ --mib-source=http://mibs.snmplabs.com/asn1/@mib@ LSI-MegaRAID-SAS-MIB
> ```
> 
> BTW, here is [the documentation](http://snmplabs.com/pysmi/mibdump.html#specifying-mib-source).

I also had the same problem, and `@mib@` trick worked for me as well, thanks. However, my pysmi version is `0.3.4`, seems to be the latest version right?

**Comment by etingof** [comment]
Yes, 0.3.4 is the latest [release](https://github.com/etingof/pysmi/releases). Are you saying that without this `@mib@` magic MIB search does not work for you?

**Comment by YorkZ** [comment]
Right, without this `@mib@` magic MIB search does not work.
